app/assets/scripts/views/project.js

```diff
@@ -223,14 +223,12 @@ var Project = React.createClass({
                   <li>{tally(data.number_served.number_served)} <small>{data.number_served.number_served_unit}</small></li>
                 </ul>
                 {disbursedFunds.loan || disbursedFunds.grant
-                  ? <ul className='inpage-stats'>
```

Comment:
@felskia the `?` here indicates that this is part of a logic block. It makes it so that that the `ul` will not be written at all when neither loans or grants are available for a project. Essentially, it says "if there are loans or grants, write a `ul`", and then "if there is a loan, write an `li` inside of it, and if there is a grant, write an `li` inside of it." Removing the top of the logic block causes a Javascript error and prevents the site from being built, and the `ul` also needs to be there structurally in order to hold the `li`s when they exist.